openshift/template.yaml

```diff
@@ -185,6 +187,11 @@ parameters:
   required: false
   name: BAYESIAN_API_HOSTNAME
 
+- description: Stack Report UI (will be of Prod, if empty)
+  displayName: Stack Report UI
+  required: false
+  name: STACK_REPORT_UI_HOSTNAME
+
```

Comment:
we are using default values in `default_config`, what's the difference..? have we adopted any new standards..?

Reply:
it is not new standard or anything, instead of having a text "Stack Report UI (will be of Prod, if empty)", express the default value over here itself.

Reply:
Ack